backend/src/jobs/userReminder.job.ts

```
@@ -0,0 +1,74 @@
+import * as cron from "node-cron";
+import NotificationService from "../services/notification.service";
+
+class UserReminderJob {
+  private static isRunning = false;
+
+  /**
+   * Check for inactive users and send reminders
+   */
+  public static async checkInactiveUsers(): Promise<void> {
+    if (this.isRunning) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)